### PR TITLE
Prepare for new tool version - don't publish yet

### PR DESCRIPTION
**Pull request by Stealth7777**

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Comment by Stealth7777** [comment]
Please do not publish yet

File: Enterprise/page-diagnostics-for-spo.md [hunk 1]
```diff
@@ -59,24 +59,23 @@ Read the following sections to learn more about the information provided in the
...
     
-3. The [**Diagnostic** tab](#how-to-use-the-diagnostic-tab) will list the rules and if any of them are marked with a red ![Cross](media/9859ac84-be43-4eae-984c-e0e827f5a228.png), then there are issues identified on the page.<br/>Each rule has its own "more information" link that you click if an item is red. That will take you to the details behind that rule and how to remediate the issue.<br/>![Diagnostics Red - Rule open](media/1598f0f7-3103-4613-8787-dfec6fffd40a.png)
+3. The [**Diagnostic tests** tab](#how-to-use-the-diagnostic-tab) will show a red section and a green section and the items in the red section will list the items that require attention. The green section will list items checked that meet thr current requirements.<br/>Each failed check has its own "Learn more" link that will take you to the details behind that rule and how to remediate the issue.<br/>![Diagnostics Red - Rule open](media/1598f0f7-3103-4613-8787-dfec6fffd40a.png)
```
Comment (nenonix) [inline Enterprise/page-diagnostics-for-spo.md]:
```suggestion
3. The [**Diagnostic tests** tab](#how-to-use-the-diagnostic-tab) will show a red section and a green section and the items in the red section will list the items that require attention. The green section will list items checked that meet the current requirements.<br/>Each failed check has its own "Learn more" link that will take you to the details behind that rule and how to remediate the issue.<br/>![Diagnostics Red - Rule open](media/1598f0f7-3103-4613-8787-dfec6fffd40a.png)
```

**Comment by mypil** [comment]
@Stealth7777 - kindly merge the commit on the copy/edit review done by nenonix.  Also, can you please provide us the date/timeframe on when you would like us to tag the doc owner for PR merging?

Thank you.

**Comment by mypil** [comment]
@Stealth7777 - We would like to follow up on this. Please assist to merge the commit on the copy/edit review done by nenonix. Also, can you please provide us the date/timeframe on when you would like us to tag the doc owner for PR merging?

Thank you.

**Comment by Stealth7777** [comment]
You can delete that merge as it is old.

**Comment by Stealth7777** [comment]
Old so can be retracted

**Pull request closed without merging**